ui/shared/components/panel/FamilyReads.jsx

```diff
@@ -0,0 +1,198 @@
+
+  // TODO better determiner of genome version?
+  const isBuild38 = igvSamples.some(sample => sample.filePath.endsWith('.cram'))
+  const genome = isBuild38 ? 'hg38' : 'hg19'
+
+  const locus = variant && getLocus(
+    variant.chrom, (!isBuild38 && variant.liftedOverPos) ? variant.liftedOverPos : variant.pos, 100,
+  )
+
+  igvTracks.push({
+    url: `https://storage.googleapis.com/seqr-reference-data/${isBuild38 ? 'GRCh38' : 'GRCh37'}/gencode/gencode.v27${isBuild38 ? '' : 'lift37'}.annotation.sorted.gtf.gz`,
+    name: `gencode ${genome}v27`,
+    displayMode: 'SQUISHED',
+  })
+
+  return {
+    locus,
+    tracks: igvTracks,
+    genome,
+    showKaryo: false,
+    showIdeogram: true,
+    showNavigation: true,
+    showRuler: true,
+    showCenterGuide: true,
+    showCursorTrackingGuide: true,
+    showCommandBar: true,
+  }
+}
+
+const ReadIconButton = props => <ButtonLink icon="options" content="SHOW READS" {...props} />
+
+const ReadButtons = React.memo(({ variant, familyGuid, igvSamplesByFamily, familiesByGuid, buttonProps, showReads }) => {
+  const familyGuids = variant ? variant.familyGuids : [familyGuid]
+
+  const familiesWithReads = familyGuids.filter(fGuid => (igvSamplesByFamily[fGuid] || []).length > 0)
+  if (!familiesWithReads.length) {
+    return null
+  }
+
+  if (familiesWithReads.length === 1) {
+    return <ReadIconButton onClick={showReads(familiesWithReads[0])} {...buttonProps} />
+  }
+
+  return [
+    <ReadIconButton key="showReads" {...buttonProps} />,
+    ...familiesWithReads.map(fGuid => (
+      <ButtonLink key={fGuid} content={`| ${familiesByGuid[fGuid].familyId}`} onClick={showReads(fGuid)} padding="0" />
+    )),
+  ]
+})
+
+ReadButtons.propTypes = {
+  variant: PropTypes.object,
+  familyGuid: PropTypes.string,
+  buttonProps: PropTypes.object,
+  familiesByGuid: PropTypes.object,
+  igvSamplesByFamily: PropTypes.object,
+  showReads: PropTypes.func,
+}
+
+const IgvPanel = React.memo(({ variant, igvSamples, individualsByGuid, hideReads }) => {
+  const igvOptions = getIgvOptions(variant, igvSamples, individualsByGuid)
+  return (
+    <Segment>
+      <ButtonLink onClick={hideReads} icon={<Icon name="remove" color="grey" />} floated="right" size="large" />
+      <VerticalSpacer height={20} />
+      <IGV igvOptions={igvOptions} />
+    </Segment>
+  )
+})
+
+IgvPanel.propTypes = {
+  variant: PropTypes.object,
+  individualsByGuid: PropTypes.object,
+  igvSamples: PropTypes.array,
+  hideReads: PropTypes.func,
+}
+
+class FamilyReads extends React.PureComponent {
+
+  static propTypes = {
+    variant: PropTypes.object,
+    layout: PropTypes.any,
+    familyGuid: PropTypes.string,
+    buttonProps: PropTypes.object,
+    familiesByGuid: PropTypes.object,
+    individualsByGuid: PropTypes.object,
+    igvSamplesByFamily: PropTypes.object,
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      openFamily: null,
+    }
+  }
+
+  showReads = familyGuid => () => {
```

Comment:
It can be more descriptive by changing this `showReads` to `getShowReadsSetter` or something like that.

Reply:
This is a function that shows the reads for the bound family. Using the words "getter" and "setter" is really not a javascript convention and changing to that would be confusing and would be a complete antipattern for how we name functions like this everywhere else. See https://github.com/broadinstitute/seqr/blob/master/ui/shared/components/table/DataTable.jsx#L82